webapp/components/PostCard/PostCard.vue

```diff
@@ -141,25 +146,37 @@ export default {
       this.$emit('unpinPost', post)
     },
   },
-}
-</script>
+  mounted() {
+    const width = this.$el.offsetWidth
+    const height = Math.min(width / this.post.imageAspectRatio, 2000)
+    const imageElement = this.$el.querySelector('.ds-card-imag')
```

Comment:
```suggestion
    const imageElement = this.$el.querySelector('.ds-card-image')
```

Reply:
Also, I'm concerned about this because we have not introduced `imageAspectRatio` for the overwhelming majority of `Post`... new `Posts` add their `imageAspectRatio`, but we would need to convert more than 18,000 `Post` where this is null... so what happens in that case?

Reply:
@mattwr18  i have change .ds-card-imag(e)

Reply:
I don't know how this code made its way into this PR – maybe from merging in `master` while we had it in there? – but it's breaking the layout. This is what happens to images without aspect ratio:
<img width="226" alt="Screenshot 2019-12-16 at 15 30 41" src="https://user-images.githubusercontent.com/11694483/70897737-3abda600-2019-11ea-89cd-debdc0d5bd66.png">

Also, weirdly, when I edit a `Post` – even without changing the picture or `blurred` status the post loses its `imageAspectRatio`. So unfortunately there's more cleaning up to do! 😞

Reply:
![cheetah-rain](https://user-images.githubusercontent.com/1324583/70970963-13f68280-20a0-11ea-88d0-b827b28972b3.gif)